openshift/templates/ecas-api/api.dc.yaml

```diff
@@ -0,0 +1,169 @@
+
+  apiVersion: template.openshift.io/v1
+  kind: Template
+  labels:
+    template: "${REPO_NAME}-template"
+  metadata:
+    name: "${REPO_NAME}-api-dc"
+    creationTimestamp:
+  objects:
+  - apiVersion: v1
+    kind: DeploymentConfig
```

Comment:
Replace with Deployment object: https://eccbc.atlassian.net/wiki/spaces/EDKB/pages/536969221/DeploymentConfig+to+Deployments